src/meltano/api/app.py

```diff
@@ -113,7 +111,7 @@ def create_app(config: dict = {}) -> Flask:  # noqa: WPS210,WPS213,B006
         logger.debug("Notifications are disabled.")
 
     # Google Analytics setup
-    tracker = LegacyTracker(project)
+    LegacyTracker(project)
```

Comment:
We should  double-check whether we even need this. I don't recall actually seeing any API endpoints fire tracking events, but its possible I also haven't noticed them.

Reply:
@WillDaSilva im pretty sure you could ditch/remove this completely.